Bottom: 5cc5b0f4462ee8a3482764473620f48e325a5f78
Top:    38096eb78f59cb2d6509bd1ba337233c431f54bc
Author: robert.berger <robert.berger@ReliableEmbeddedSystems.com>
Date:   2020-02-13 19:45:26 +0000

remove gcc-9 complaints: -Wno-error=format-overflow

Signed-off-by: student-lxc-yp-builder-2 <student-lxc-yp-builder-2@ReliableEmbeddedSystems.com>


---

diff --git a/recipes-core/icedtea/icedtea7-native.inc b/recipes-core/icedtea/icedtea7-native.inc
index 8d0dc71..e9b7b11 100644
--- a/recipes-core/icedtea/icedtea7-native.inc
+++ b/recipes-core/icedtea/icedtea7-native.inc
@@ -26,7 +26,8 @@ CXXFLAGS_append = " -fno-tree-dse"
 CXX_append = " -std=gnu++98"
 
 # WORKAROUND: ignore errors from new compilers
-CFLAGS_append = " -Wno-error=stringop-overflow -Wno-error=return-type"
+#CFLAGS_append = " -Wno-error=stringop-overflow -Wno-error=return-type"
+CFLAGS_append = " -Wno-error=stringop-overflow -Wno-error=return-type -Wno-error=format-overflow"
 
 inherit native java autotools pkgconfig
 inherit openjdk-build-helper
diff --git a/recipes-core/openjdk/openjdk-8-common.inc b/recipes-core/openjdk/openjdk-8-common.inc
index d8b30b8..430e947 100644
--- a/recipes-core/openjdk/openjdk-8-common.inc
+++ b/recipes-core/openjdk/openjdk-8-common.inc
@@ -175,7 +175,7 @@ EXTRA_OECONF_append = "\
 FLAGS_GCC6 = "-fno-lifetime-dse -fno-delete-null-pointer-checks"
 FLAGS_GCC7 = "-fno-lifetime-dse -fno-delete-null-pointer-checks"
 FLAGS_GCC8 = "-fno-lifetime-dse -fno-delete-null-pointer-checks"
-FLAGS_GCC9 = "-fno-lifetime-dse -fno-delete-null-pointer-checks"
+FLAGS_GCC9 = "-fno-lifetime-dse -fno-delete-null-pointer-checks -Wno-error=format-overflow"
 
 # flags for -native, and for bits that need a host-tool during -cross
 BUILD_CFLAGS_append = " ${@openjdk_build_helper_get_build_cflags(d)}"
